### PR TITLE
[pendulum_control Install targets to project lib

### LINKED ISSUE #622: pendulum_control executables not available via ros2 run
See https://github.com/osrf/ros2_test_cases/issues/673#issuecomment-1540153152

### PR DESCRIPTION
**Pull request by Yadunund**
Fix #622 

When testing locally tag autocomplete shows the available executables and I was able to run them too 
```
yadunund@ubuntu-22-04:~/ros2_rolling$ ros2 run pendulum_control pendulum_
pendulum_demo         pendulum_launch.bash  pendulum_logger       pendulum_teleop
```

```
yadunund@ubuntu-22-04:~/ros2_rolling$ ros2 run pendulum_control pendulum_demo 
Couldn't set scheduling priority and policy: Operation not permitted
Initial major pagefaults: 0
Initial minor pagefaults: 2108627
No results filename given, not writing results
rttest statistics:
  - Minor pagefaults: 124
  - Major pagefaults: 0
  Latency (time after deadline was missed):
    - Min: 66848 ns
    - Max: 112908 ns
    - Mean: 73036.601000 ns
    - Standard deviation: 4056.924466
```

### CODE REVIEW

File: pendulum_control/CMakeLists.txt [hunk 1]
```diff
@@ -57,11 +57,11 @@ install(TARGETS
...
   pendulum_logger
   pendulum_teleop
-  DESTINATION bin)
+  DESTINATION lib/${PROJECT_NAME})
 
 install(
   PROGRAMS "scripts/pendulum_launch.bash"
-  DESTINATION bin
+  DESTINATION lib/${PROJECT_NAME}
 )
```
[the remark this answers is not in the document]
Comment (Yadunund) [reply pendulum_control/CMakeLists.txt]: Gotcha [3692296](https://github.com/ros2/demos/pull/624/commits/369229617360b4c8ddc9de39e76ea13c02d1843e)

**Review by clalancette — APPROVED** [review]

**Comment by clalancette** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/18727/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/13253/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/19465/)

**Comment by Yadunund** [comment]
Do we backport to iron?

**Comment by clalancette** [comment]
> Do we backport to iron?

I don't feel like this is an important fix, so at this point I would suggest no.

**Pull request merged**